### DISCUSSION TITLE
File Description - CDE 11280338

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#54] [Metadata Field Ratification]
caDSR Link: [11280338](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=11280338%20and%20ver_nr=1)

Definition: A free text field that can be used to document the content and other details about an electronic file that may not be captured elsewhere.

Discussion:  Should the Federation add this as a required field and allow each member to determine what to enter?  Should there be further harmonization requirements for the text entered in this field?

**Comment by calkinsh** [comment]
I don't think this should be a required field- we have many cases where we would not have anything to populate here.  If it is required I suspect in those cases we might just populate it with repeated info such as the file name which defeats the purpose. So my recommendation would be to make this an optional or recommended element but not required.

**Comment by Ssandor13** [comment]
St. Jude also agrees this should be an optional field.

**Comment by GeoffLyle** [comment]
A possible solution could be to have this as an optional field, and if a file description is added, it would use a harmonized field name (ex. **file_description**).
While it doesn't have to implemented immediately, a nice feature would be the ability to search the free text fields associated with the file.  A researcher interested in gene expression could search for 'expression' and find files of interest.

**Comment by GeoffLyle** [comment]
CDE accepted at January 25th meeting.

CDE will not be required, but will have a harmonized field name for users to extract information.  Decision on how to return a _null_ value for this field when it is not populated has been left to the API implementation group.
